test/fixtures/comments.mdx

```diff
@@ -0,0 +1 @@
+<!-- eslint-disable-next-line no-console -- Here's a description about why this configuration is necessary. -->
```

Comment:
@jablko 

Can you add a true expression to show that the comment is really working?

Reply:
Good point -- what do you think of this? I added `reportUnusedDisableDirectives: 'error'` and confirmed that there's an unused configuration comment after this regexp fix, where before it there's none:
```diff
  ● fixtures › should match all snapshots
    expect(received).toMatchSnapshot(hint)

    Snapshot name: `fixtures should match all snapshots: comments.mdx 1`

    - Snapshot  - 10
    + Received  +  1

    - Array [
    -   Object {
    -     "column": 2,
    -     "line": 1,
    -     "message": "Unused eslint-disable directive (no problems were reported from 'no-console').",                                                          
    -     "nodeType": null,
    -     "ruleId": null,
    -     "severity": 2,
    -   },
    - ]
    + Array []

      43 |     const results = await getCli().lintFiles('test/fixtures/*.{md,mdx}')                                                                             
      44 |     for (const { filePath, messages } of results) {
    > 45 |       expect(messages).toMatchSnapshot(path.basename(filePath))
         |                        ^
      46 |     }
      47 |   })
      48 |

      at test/fixtures.test.ts:45:24
      at fulfilled (node_modules/tslib/tslib.js:114:62)

Reply:
LGTM